internal/distro/fedora/images.go

```diff
@@ -48,6 +49,7 @@ func osCustomizations(
 	osc.ExtraBaseRepos = osPackageSet.Repositories
 
 	osc.Containers = containers
+	osc.CustomRepos = customRepos
```

Comment:
You should take this from passed blueprint customizations instead, not as a separate function argument.